bin/updwfmet.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Give this script a longer name. It is impossible to understand what this abbreviation means.

Reply:
Tried to fit 8 characters, hm, yes, quite an outdated habit.